syntax = "proto3";

message ErrorMessage {
    uint32 return = 1;
}
